api_spec.yaml

```diff
@@ -1743,6 +1747,8 @@ paths:
                   $ref: '#/components/schemas/token'
                 userDataInJWT:
                   $ref: '#/components/schemas/userDataInJWT'
+                grants:
+                  $ref: '#/components/schemas/grantPayload'
       responses:
```

Comment:
Do we also need to API to just update the grant based on the sessionHandle only? Simlar to how we have `/recipe/jwt/data` PUT?

Reply:
I've added one, but that would only be reflected after a session refresh, so I'm not sure how useful that is.

Reply:
It's there more for completeness of APIs.